lib/web_ui/lib/src/engine/dom_canvas.dart

```diff
@@ -135,6 +135,17 @@ class DomCanvas extends EngineCanvas with SaveElementStackTracking {
   }
 }
 
+/// Converts a shadow color specified by the framework to the color that should
+/// actually be applied when rendering the element.
+///
+/// Returns a color for box-shadow based on blur filter at sigma.
+ui.Color blurColor(ui.Color color, double sigma) {
+  final double strength = math.max(math.min(
+      math.sqrt(sigma) / (math.pi * 2.0), 1.0), 0.0);
```

Comment:
Do we need `math.max`? Presumably sigma is >= 0, and so `math.min(math.sqrt(sigma) / (math.pi * 2.0), 1.0)` is also >= 0.

Reply:
Thx. Done.